mm2src/mm2_bitcoin/chain/src/block_header.rs

```diff
@@ -1034,6 +1037,437 @@ mod tests {
         assert_eq!(serialized.take(), headers_bytes);
     }
 
+    #[test]
+    fn test_nmc_block_headers_serde_11() {
+        // NMC block headers
+        // start - #622807
+        // end - #622796
+        // Ref: https://chainz.cryptoid.info/nmc/block.dws?622807.htm
+        let headers_bytes: &[u8] = &[
```

Comment:
Could you please move these bytes to a file.
An example:
https://github.com/KomodoPlatform/atomicDEX-API/blob/09a91e0cd4a80657c1b4a4428a61e7145474cec7/mm2src/mm2_main/src/ordermatch_tests.rs#L3177
But unlike it, you can use [include_bytes](https://doc.rust-lang.org/std/macro.include_bytes.html)

Reply:
Ok cool thanks, would it be required to do this for other tests in `block_header`?

Reply:
I think it would be great!

Reply:
done

Reply:
Good walkaround!